src/components/admin/adminComponents/Information.tsx

```diff
@@ -1,17 +1,54 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import { Flex, FormControl, SimpleGrid, Text, useColorModeValue, Input, FormLabel, Box, Button, Card } from '@chakra-ui/react';
 import useTranslation from 'next-translate/useTranslation';
 import { isEventsMate } from '../../../utils/orientation';
+import { api } from '~/utils/api';
+import { UserData } from '~/interfaces/user';
 
 interface InformationProps {
-  [key: string]: any;
+  user: UserData
 }
 
-const Information: FC<InformationProps> = () => {
+const Information: FC<InformationProps> = ({ user }) => {
   const textColorPrimary = useColorModeValue('secondaryGray.900', 'white');
   const textColorSecondary = 'secondaryGray.600';
-  const { t } = useTranslation()
-  
+  const { t } = useTranslation();
+
+  const [username, setUsername] = useState(user.username || '');
+  const [email, setEmail] = useState(user.email || '');
+  const [firstName, setFirstName] = useState( '');
+  const [lastName, setLastName] = useState( '');
+  const [userSettings, setUserSettings] = useState<UserData>()
+
+
+  const fetchUserSettings = async () => {
+    try {
+      const { data } = await api.get<UserData>('users/settings');
+      setUserSettings(data);
+    } catch (error) {
+      console.error('Error fetching user settings:', error);
+    }
+  };
+
+  const saveChanges = async () => {
+    const payload = {
+      email,
+      username,
+      firstName,
+      lastName,
+    };
+
+    try {
+      await api.put('users/', payload);
+      alert('Profile updated successfully');
```

Comment:
místo alertu použít notification store, je to ted zustand ale funguje to stejne v podstate